common/logger.go

```diff
@@ -25,7 +25,9 @@ func ConcatWithRequest(data M, req Request) M {
 		"method":     req["method"],
 	}
 	for header, values := range req["headers"].(http.Header) {
-		kvData[header] = strings.Join(values, ";")
+		if header != "Authorization" {
```

Comment:
I sanity checked that these headers are all normalized (capitalized appropriately) and Go HTTP lib does that for us - https://pkg.go.dev/net/http#Request - , so just checking against the constant `Authorization` works!

Reply:
Yep, I tested with spew.Dump() to make sure I was getting the headers I expected